frame/balances/src/migration.rs

```diff
@@ -10,7 +10,7 @@ pub use try_runtime::*;
 // --- darwinia-network ---
 use crate::*;
 
-pub fn migrate<T, I>()
+pub fn migrate_lock<T, I>()
```

Comment:
Prefer to keep this.
And I don't need to change on darwinia side.

Reply:
I have thought about this so I just rename it. We might have other migrations in the future, it's better to keep the migration history?

Reply:
I think it isn't the time to do that.
If a lot of teams are using our pallet, I would consider that.